.github/workflows/simulator-nightly.yml

```diff
@@ -37,9 +37,34 @@ jobs:
       - uses: taiki-e/install-action@nextest
       - name: cargo simtest
         run: |
-          export MSIM_TEST_SEED=$(date +%s)
+          # TODO: If possible, if we build a docker image, and then run the below work inside the docker image,
+          # that would be great. The reason is that when failing seeds are found, they will only repro on linux,
+          # not mac. Therefore, to debug the failures, it would be excellent if you could just pull down the
+          # docker image that the failure happened in, so it could be reproduced locally very easily.
```

Comment:
locally to the mac?  linux amd64 won't work there unless you're running some virtual machine with that architecture.

Reply:
Docker desktop works great.

Reply:
(it uses qemu if i'm not mistaken)